### PR TITLE
should leave out Cargo.toml `documentation` and `homepage` because defaults are better

### PR DESCRIPTION
**Pull request by Connoropolous**
## PR summary

It'd be preferable if `documentation` on the crates.io page actually linked you to https://docs.rs/hdk/0.0.40-alpha1/hdk/
https://crates.io/crates/hdk/0.0.40-alpha1

Right now it takes you back to github.
This is true of other crates in this repo as well

## testing/benchmarking notes

( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )

## followups

( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )

## changelog

- [ ] if this is a code change that effects some consumer (e.g. zome developers) of holochain core,  then it has been added to [our between-release changelog](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG-UNRELEASED.md) with the format 

```markdown
- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)
```

## documentation

- [ ] this code has been documented according to our [docs checklist](https://hackmd.io/@freesig/Hk9AmKJNS)

### CODE REVIEW

File: crates/hdk/Cargo.toml [hunk 1]
```diff
@@ -4,8 +4,6 @@ edition = "2018"
 version = "0.0.40-alpha1"
 description = "holochain hdk"
 license = "GPL-3.0-only"
-homepage = "https://github.com/holochain/holochain-rust"
```
Comment (lucksus) [inline crates/hdk/Cargo.toml]: Should this be `https://holochain.org`?

**Comment by Connoropolous** [comment]
I'm confused... some have 'homepage' and 'documentation', others have 'repository' and 'documentation'. all seem to have, and properly require 'documentation'. It appears as if the other two are optional, given that we're seeing ones without both of those.

I discoved in holochain-persistence a good example of linking to docs.rs. I thought it meant just leaving it off and it defaulted, but it looks like it has to be set:
https://github.com/holochain/holochain-persistence/blob/develop/crates/holochain_persistence_api/Cargo.toml#L11-L12
These seem like very reasonable settings:
```toml
documentation = "https://docs.rs/holochain_persistence_api"
repository = "https://github.com/holochain/holochain-persistence"
```

Screen Shot 2019-12-07 at 9 25 58 AM

**Comment by thedavidmeister** [comment]
https://github.com/holochain/holochain-rust/pull/1911 will show whether it works or not so i'll try to wrap that up quickly

**Comment by thedavidmeister** [comment]
@Connoropolous ignore what i said, the publishing dry run approach is fundamentally broken due to dependency management, lets ship this

**Comment by Connoropolous** [comment]
Great. Granted this works it should be done in many other crates too

**Pull request merged**